lib/mongoid/enum/configuration.rb

```diff
@@ -0,0 +1,19 @@
+module Mongoid
+  module Enum
+    class Configuration
+      attr_accessor :field_name_prefix
+
+      def initialize
+        self.field_name_prefix = '_'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.